src/main/java/seedu/revision/ui/MainWindow.java

```diff
@@ -71,6 +71,32 @@ public void handleStart(Mode mode) throws CommandException {
         }
     }
 
+    /**
+     * Shows history of results as a line graph. Loads the window components for line graph.
+     * @throws CommandException
+     */
+    @FXML
+    public void handleHistory() throws CommandException {
+        if (this.logic.getStatisticsList().size() > 0) {
+
```

Comment:
What's supposed to be in this if statement ah?

Reply:
supposed to show line graph, still task ongoing